src/meta/src/hummock/manager/compaction.rs

```diff
@@ -32,6 +35,52 @@ pub struct Compaction {
     pub compaction_statuses: BTreeMap<CompactionGroupId, CompactStatus>,
 }
 
+/// Cancels all tasks assigned to `context_id`.
+pub fn cancel_all_assigned_tasks(
```

Comment:
disscuss with zwang, we can remove `compact_statuses` and `compact_task_assignment` input.

Reply:
Thanks, nice suggestion.